azure-pipelines.yml

```diff
@@ -1,71 +1,73 @@
-# ASP.NET Core
```

Comment:
Can you check if the newlines got changed for this file or something? It would help to just have the changed lines in the diff.

Reply:
I honestly have no idea what causes this, or how to check newline usage. When I look at the diff in GitHub's desktop client, it only shows lines 54-55 having differences. WinMerge, my usual diff tool, also points to lines 54-55.

Reply:
My initial commit shows a proper diff, but the lowercase g fix seems to have messed it up. I may have committed this change directly from the browser.

First link shows the two lines added, second link shows all text as being replaced but also highlights the actual fix on those new lines.
https://github.com/InsaneFirebat/sm_practice_hack/commit/c16e6daa20a51d0d3c0566456d6af686d991fde6#diff-fec826feae04e51c0d94076385408bdc
https://github.com/InsaneFirebat/sm_practice_hack/commit/bf04e4f12683fbae25a6666d9f18f0b169e28934#diff-fec826feae04e51c0d94076385408bdc